tests/queries/0_stateless/01548_query_log_query_execution_ms.sql

```diff
@@ -0,0 +1,28 @@
+set log_query_threads=1;
+set log_queries_min_type='QUERY_FINISH';
+set log_queries=1;
+select '01548_query_log_query_execution_ms', sleep(0.4) format Null;
+set log_queries=0;
+set log_query_threads=0;
+
+system flush logs;
+
+select count()
+from system.query_log
+where
+    query like '%01548_query_log_query_execution_ms%'
+    and current_database = currentDatabase()
+    and query_duration_ms between 100 and 800
```

Comment:
A query can become arbitrarily long under the load... let's see how flaky it's going to be, if it's too bad we can remove this test.